doc/api/errors.md

```diff
@@ -459,10 +459,27 @@ Returns a number corresponding to the **negated** error code, which may be
 referenced in `man 2 intro`. For example, an `ENOENT` error has an `errno` of
 `-2` because the error code for `ENOENT` is `2`.
 
+In some mudules, returns the same string as `error.code`.
+
 #### error.syscall
 
 Returns a string describing the [syscall][] that failed.
 
+#### error.path
+
+Returns a string of the specified invalid pathname in case of using some modules
```

Comment:
Even though there are existing contrary examples in this very file, I think we should avoid saying `returns` when talking about a property. We should reserve `returns` for functions.

So, instead of `Returns a string that...`, just `A string that...`. Yes, it will technically be a sentence fragment, but I think it's acceptable in this case. It would certainly be clear and understandable which is the most important thing. 

/cc @nodejs/documentation